app/src/main/res/drawable/google_g.xml

```diff
@@ -0,0 +1,15 @@
+<vector android:height="24dp" android:viewportHeight="46.0"
+    android:viewportWidth="46.0" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#4285f4"
```

Comment:
This is google logo (newer, with four colors), we wouldn't need to change its color. If needed we will change logo itself.